docs_src/index.md

```diff
@@ -1,34 +1,25 @@
-# Vision Self Checkout
+# Automated Self Checkout
 
-## Introduction
-This guide helps you build and run the Vision Self Checkout solution.
+## Overview
 
-Upon completing the steps in this guide, you will be ready to run and benchmark pipelines on different hardware setups.
+The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy a self-checkout use case using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self checkout pipeline that is optimized for Intel® hardware.  The reference solution also includes a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per workload.
 
-### Overview
+[![Automated Self Checkout Diagram](./images/vision-checkout-1.0.png)](./images/vision-checkout-1.0.png)
 
-The Vision Self Checkout solution is a set of pre-configured pipelines that are optimized for performance on Intel. The pipelines run several models including yolov5s, efficientnet-b0, horizontal-test-detection-002, and text-recognitiion-0012-gpu. Details about the pipelines and how to run them can be found [HERE](./pipelinesetup.md)
+## Prerequisites
 
-A set of branchmarking tools have been provided to demonstrate the performance of the pipelines on Intel hardware. Once you have completed the [pipeline setup steps](./pipelinesetup.md) you will be able to run benchmark by following these [steps](./benchmark.md) 
+The following items are required to build the Automated Self Checkout solution. You will need to follow the guide that matches your specific hardware setup.
 
-[![Vision Self Checkout Diagram](./images/vision-checkout-1.0.png)](./images/vision-checkout-1.0.png)
+- [Ubuntu LTS Boot Device](https://ubuntu.com/tutorials/install-ubuntu-desktop#3-create-a-bootable-usb-stick)
+- [Docker](https://docs.docker.com/engine/install/ubuntu/)
+- [Git](https://git-scm.com/download/linux)
 
-### Prerequisites
+## Platform Installation
 
-The following items are required to build the Vision Self Checkout solution. You will need to follow the guide that matches your specific hardware setup.
+First verify that your platform is included on the [supported platform list](./platforms.md)
 
-- Ubuntu LTS Boot Device
-- Docker
-- GIT
+Once verified the setup steps for each supported hardware platform can be found [HERE](./hardwaresetup.md).
```

Comment:
Replace "Once verified the setup steps..." with "Verify that the setup steps..."